pwndbg/commands/ai.py

```diff
@@ -186,42 +221,72 @@ def build_prompt(question, command=None):
 {source}
 ```
 """
-    return finish_prompt(prompt, question)
+    return strip_colors(prompt)
 
 
-def build_prompt_from_command(question, command):
+def build_command_prompt_body(command):
     prompt = (
         f"""Running the command `{command}` in the GDB debugger yields the following output:\n"""
     )
     output = gdb.execute(command, to_string=True)
     print(output)
     prompt += f"""\n```\n{output}\n```\n\n"""
-    return finish_prompt(prompt, question)
+    return strip_colors(prompt)
 
 
 def strip_colors(text):
     ## Now remove all ANSI color codes from the prompt
     return re.sub(r"\x1b[^m]*m", "", text)
 
 
-def finish_prompt(prompt, question):
-    ## If the context hasn't changed, include the last question and answer
-    ## (we could add more of these, but there are length limitations on prompts)
-    for (q, a) in zip(last_question, last_answer):
-        prompt += f"""Question: {q}\n\nAnswer: {a}\n\n"""
-
-    prompt += f"""Question: {question}
-
-Answer: """
-
-    prompt = strip_colors(prompt)
-
-    return prompt
+def query_openai_chat(prompt, model="gpt-3.5-turbo", max_tokens=100, temperature=0.0):
+    if verbosity > 0:
+        print(
+            M.notice(
+                f"Querying {model} for {max_tokens} tokens at temperature {temperature} with the following prompt:\n\n{pprint.pformat(prompt)}"
+            )
+        )
+    data = {
+        "model": model,
+        "max_tokens": max_tokens,
+        "messages": prompt,
+        "temperature": temperature,
+    }
+    host = "api.openai.com"
+    path = "/v1/chat/completions"
+    url = f"https://{host}{path}"
+    r = requests.post(
+        url,
```

Comment:
```suggestion
        f"https://api.openai.com/v1/chat/completions",
```
It seems we don't use the `host`, `path` and `url` variables any further, so they can be deleted and the full url hardcoded here

Reply:
(same in other places)